print.html

```diff
@@ -189,7 +189,10 @@ <h1>Table of Contents</h1>
       <li>Standard</a>
```

Comment:
It might make more sense to break these out in the same way they were in `index.md`, so not under one heading named 'Standard' but one for community and one for codebase criteria.

Reply:
agreed